mujoco_warp/_src/solver.py

```diff
@@ -545,17 +545,17 @@ def _next_quad(
 
     alpha = lo_next_alpha[worldid]
     # TODO(team): active and conditionally active constraints
-    if jaref + alpha * jv < 0.0:
+    if efcid < d.ne[0] or jaref + alpha * jv < 0.0:
```

Comment:
it should be more performant to read the value of `ne` from global memory and then utilize a local variable. something like:
````
ne = d.ne[0]
...
efcid < ne
```